lib/fuse_versionscript

```diff
@@ -200,6 +200,7 @@ FUSE_3.17 {
 		fuse_set_fail_signal_handlers;
 		fuse_log_enable_syslog;
 		fuse_log_close_syslog;
+		fuse_add_kernel_mount_opt;
```

Comment:
That needs to go into a new 3.18 section. See here
https://github.com/libfuse/libfuse/pull/1177/commits/ab33d7f8dcad1c20577f5461d790e3f9b497af14

('m still reviewing and cleaning it up myself)